### PR TITLE
Update the overview to get the numbering/bullet points right

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
Tweaks @martipos PR:

- removes the leading "> " on the overview (I don't think it should be "quoted" text, which is what that implies).
- Always uses 4 spaces for indenting bullet levels.  GitHub markdown only needs 2, but the Markdown that mkdocs uses REQUIRES 4.
- Some editorial tweaks to the overview - particularly around `entryHash` and `versionId`

File: docs/overview.md [hunk 1]
```diff
@@ -30,88 +30,85 @@ as both `did:web` and `did:tdw`. These possibilities carter to a flexible and br
...
->       - The SCID is only calculated for the very first DID Log Entry and not for consecutive DID updates (i.e., new DID Log Entry)
->       - Instead of a SCID, each a new EntryHash is calculated in a similar way, for every consecutive DID Log Entry
->            - EntryHash := `base58btc(multihash(JCS(entry), <hash algorithm>))`
-> - Note: Both the SCID and the EntryHash are calculated *before* the DI proof calculation
-> - `did:tdw` uses the same DID-to-HTTPS transformation as `did:web`, so `did:tdw`'s  `did.jsonl` (JSON Lines) file is found in the same location as `did:web`'s `did.json` file, and supports an easy transition from `did:web` to gain the added benefits of `did:tdw`.
-> - For backwards compatibility, and for verifiers that "trust" `did:web`, a
+4. Calculate the data integrity (DI) proof
+    - `proof` := a proof calculated across the *entire* DID Log Entry and signed with an `updateKeys` (and optionally by witnesses). Values of required attributes include:
+        - `type` := eddsa-jcs-2022
+        - `proofPurpose` := `assertionMethod`
```
Comment (PatStLouis) [inline docs/overview.md:85]:
```suggestion
        - `type` := `DataIntegrityProof`
        - `cryptosuite` := `eddsa-jcs-2022`
        - `proofPurpose` := `assertionMethod`
```

**Review by PatStLouis — CHANGES_REQUESTED** [review]

File: docs/overview.md (hunk 1 above)
Comment (swcurran) [reply docs/overview.md]: Fixed.

**Review by PatStLouis — APPROVED** [review]

**Comment by swcurran** [comment]
Thx!

**Pull request merged**